src/ScriptExecutor.php

```diff
@@ -43,7 +43,8 @@ public function __construct(Composer $composer, IOInterface $io, Options $option
      */
     public function execute(string $type, string $cmd): void
     {
-        if (null === $expandedCmd = $this->expandCmd($type, $this->options->expandTargetDir($cmd))) {
+        $translatedCmd = $this->options->expandTargetDir($cmd);
```

Comment:
@ricardocino thanks for providing this feature. My only (minor) concern is the `$translatedCmd` variable name. "Translated" has a strong meaning unrelated to this feature, so it may be confusing. Sadly I cannot propose a better name ($parsedCm, $resolvedCmd, $realCmd, etc. don't look good either to me).